src/server/game/Entities/Object/Object.h

```diff
@@ -487,6 +487,7 @@ class WorldObject : public Object, public WorldLocation
         bool IsWithinDistInMap(WorldObject const* obj, float dist2compare, bool is3D = true) const;
         bool IsWithinLOS(float x, float y, float z) const;
         bool IsWithinLOSInMap(WorldObject const* obj) const;
+        virtual Position GetHitSpherePointFor(Position const dest) const;
```

Comment:
It looked cleaner to me like that, for players I intend(ed) to use ```GetPositionZ() + Player::GetCollisionHeight```. Instead of + 2.0f
I could do it in one function, but won't it be too messy if I do so ? :/